Issue with test for rank_size

@bktaha would you have any time to look into this issue with the rank_size test? 

`Travis` was broken due to the migration to `.com` from the `.org` site and QuantEcon.py was using legacy connections -- which has now been fixed. 

I'll hold off on issuing a new release until this is fixed. 

```
FAIL: Tests `rank_size` function, which generates rank-size data for
896----------------------------------------------------------------------
897Traceback (most recent call last):
898  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/nose/case.py", line 198, in runTest
899    self.test(*self.arg)
900  File "/home/travis/build/QuantEcon/QuantEcon.py/quantecon/tests/test_inequality.py", line 119, in test_rank_size
901    assert_allclose(r_sqval, 1, rtol=1e-4)
902  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/numpy/testing/_private/utils.py", line 1528, in assert_allclose
903    verbose=verbose, header=header, equal_nan=equal_nan)
904  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/numpy/testing/_private/utils.py", line 840, in assert_array_compare
905    raise AssertionError(msg)
906AssertionError: 
907Not equal to tolerance rtol=0.0001, atol=0
908
909Mismatched elements: 1 / 1 (100%)
910Max absolute difference: 0.00188538
911Max relative difference: 0.00188538
912 x: array(0.998115)
913 y: array(1)
````

--- a/quantecon/tests/test_inequality.py
+++ b/quantecon/tests/test_inequality.py
@@ -103,7 +103,8 @@ def test_rank_size():
     of the size of the distribution.
     """
 
-    sample_size = 1000
+    np.random.seed(15)
+    sample_size = 10000
     c = 0.74
 
     # Tests Pareto; r_squared ~ 1
@@ -116,10 +117,9 @@ def test_rank_size():
     _, _, r_value, _, _ = linregress(np.log(rank_data), np.log(size_data))
     r_sqval = r_value**2
 
-    assert_allclose(r_sqval, 1, rtol=1e-4)
+    assert_allclose(r_sqval, 1, rtol=1e-3)
 
     # Tests Exponential; r_squared < 1
-    np.random.seed(13)
     z = np.random.randn(sample_size)
 
     exp_draw = np.exp(z)
@@ -129,6 +129,5 @@ def test_rank_size():
                                          np.log(size_data_exp))
     r_sqval_exp = r_value_exp**2
 
-    assert_raises(AssertionError, assert_allclose, r_sqval_exp, 1, rtol=1e-4)
-
+    assert_raises(AssertionError, assert_allclose, r_sqval_exp, 1, rtol=1e-3)
 

